sdk/search/Microsoft.Azure.Search/tests/Microsoft.Azure.Search.Tests.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.5.1" />
```

Comment:
We should also try not to override here.

Reply:
The version that was referenced automatically was 4.5.0, and guidance is to use 4.5.1 for 4.0.0 - 4.5.0. 4.7.2 will update many more references than we want. The props file has 4.7.2 which is not a good replacement for 4.5.0.